apps/condo/domains/miniapp/schema/B2BAppNewsSharingConfig.js

```diff
@@ -61,6 +77,12 @@ const B2BAppNewsSharingConfig = new GQLListSchema('B2BAppNewsSharingConfig', {
             isRequired: false,
         },
     },
+    hooks: {
+        afterChange: async ({ updatedItem, listKey }) => {
+            await iconMetaAfterChange({ updatedItem, listKey })
+            await previewPictureMetaAfterChange({ updatedItem, listKey })
+        },
```

Comment:
Should this hooks run after each change of that model?

Reply:
Yes, i copied the logic from b2bapp.js